docs/feature/index/index.md

```diff
@@ -0,0 +1,169 @@
+documents by default, yielding instant query performance on everything.
+
+:::{rubric} About
+:::
+By default, CrateDB indexes all data in every field, and each indexed field has
+a dedicated, optimized data structure. For example, text fields are stored in
+inverted indices, and numeric and geo fields are stored in BKD trees.
+
+The ability to use the per-field data structures to assemble and return search
+results is what makes CrateDB so fast.
+
+:::{rubric} Details
+:::
+For a quick refresh about the technologies behind the storage engine of CrateDB,
+let us refer you to to a few upstream documentations and articles about Lucene
+and Elasticsearch.
+
+- [Searching and Indexing With Apache Lucene]
+- [An Introduction to Elasticsearch]
+- [Elasticsearch for Dummies]
+- [Elasticsearch: Documents and Indices]
+
+See also an article by Rockset, which refers to the same powerful indexing
+regime, claiming that paradigm would be a unique invention.
+
+- [Converged Index™: The Secret Sauce Behind Rockset's Fast Queries]
+
+On disk, CrateDB stores data into Lucene indexes. By default, all fields are indexed,
+nested or not, but the indexing can be turned off selectively.
+
+- {ref}`storage-layer`
+::::
+
+
+::::{grid-item}
+:class: rubric-slim
+:columns: auto 3 3 3
+
+:::{rubric} Reference Manual
+:::
+- {ref}`crate-reference:fulltext-indices`
+- {ref}`crate-reference:type-geo_shape-index`
+
+{tags-primary}`Index Types`
+{tags-primary}`Index Everything`
+{tags-primary}`Fast Query Execution`
+::::
+
+:::::
+
+
+
+## Usage
+
+Handling data types in the most efficient way, for maximum usability, is built
+into CrateDB. You automatically leverage its indexing data structures by
+submitting SQL queries to the execution engine.
+
+
+## Learn
+
+Articles about CrateDB's uniqueness as an "index everything by default"
+database, insights into the technologies behind, and also comparing it
+with solutions from other vendors.
+
+
+
+::::{info-card}
+:::{grid-item}
+:columns: auto 9 9 9
+**Blog: Indexing and Storage in CrateDB**
+
+{{ '{}[Indexing and Storage in CrateDB]'.format(blog) }}
+
+Learn about the fundamentals of the CrateDB storage layer,
+looking at the three main Lucene structures that are used within CrateDB:
+Inverted Indexes for text values, BKD-trees for numeric values, and Doc Values.
+:::
+:::{grid-item}
+:columns: auto 3 3 3
+{tags-primary}`Fundamentals` \
+{tags-secondary}`Converged Indexing`
+{tags-secondary}`Deep Dive`
+:::
+::::
+
+
+::::{info-card}
+:::{grid-item}
+:columns: auto 9 9 9
+**Blog: Time Series Benchmark on CrateDB and MongoDB**
+
+{{ '{}[Time Series Benchmark on CrateDB and MongoDB]'.format(blog) }} {{ '{}[Independent comparison of CrateDB and MongoDB using Time Series Benchmark Suite]'.format(readmore) }}
+
+> When using CrateDB – that got started as a project around those times [^es-advent] –
+> it's like you've stumbled into an alternative reality where Elastic is a
+> proper database.
```

Comment:
What do you think about shortening your quote like this, specifically for publishing on this page, @henrikingo?
```suggestion
> When using CrateDB,
> it's like you've stumbled into an alternative reality where Elastic is a
> proper database.
```